<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="summaryPage">
     <attribute name="title">
      <string>Yhteenveto</string>
     </attribute>
     <widget class="QTableWidget" name="meatSharedTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>501</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="meatSharedLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jaetut lihat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshPushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Päivitä</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="killsPage">
     <attribute name="title">
      <string>Kaadot</string>
     </attribute>
     <widget class="QDateEdit" name="shotDateEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaatopäivä</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ampuja</string>
      </property>
     </widget>
     <widget class="QComboBox" name="shotByComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addShotPushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lisää</string>
      </property>
     </widget>
     <widget class="QComboBox" name="shotByComboBox2Value">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="showIdPushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Näytä ID</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTietokanta">
    <property name="title">
     <string>Tietokanta</string>
    </property>
    <addaction name="actionServerSettings"/>
   </widget>
   <widget class="QMenu" name="menuOhje">
    <property name="title">
     <string>Ohje</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTietokanta"/>
   <addaction name="menuOhje"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionServerSettings">
   <property name="text">
    <string>Palvelinasetukset...</string>
   </property>
   <property name="toolTip">
    <string>Muuta tietokantapalvelimen asetuksia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Ohjekirja</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Tekijät</string>
   </property>
  </action>
  <action name="actionRoolit">
   <property name="text">
    <string>Roolit</string>
   </property>
  </action>
  <action name="actionLukitse_asetukset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lukitse asetukset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
